src/styles/blocks/_nav.scss

```diff
@@ -0,0 +1,50 @@
+
+.logo {
+  display: flex;
+
+  &__image {
+    display: flex;
+    height: 40px;
+  }
+}
```

Comment:
It's a separate block, so it's styles should be in separate file. But it's better to create BEM element from it instead of block for logo